objax/jaxboard.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Seems you added an empty line on this file for some reason.

Reply:
sorry. didn't see that one.